wagtail/core/models.py

```diff
@@ -2060,21 +2060,40 @@ def move(self, target, pos=None, user=None):
         )
 
         # Log
-        PageLogEntry.objects.log_action(
-            instance=self,
-            action='wagtail.move',
-            user=user,
-            data={
-                'source': {
-                    'id': parent_before.id,
-                    'title': parent_before.specific_deferred.get_admin_display_title()
-                },
-                'destination': {
-                    'id': parent_after.id,
-                    'title': parent_after.specific_deferred.get_admin_display_title()
+        # Page was reordered because it's parent did not change
+        if parent_before.id == target.id:
+            PageLogEntry.objects.log_action(
+                instance=self,
+                action='wagtail.reorder',
+                user=user,
+                data={
+                    'source': {
+                        'id': parent_before.id,
+                        'title': parent_before.specific_deferred.get_admin_display_title()
+                    },
+                    'destination': {
+                        'id': parent_after.id,
+                        'title': parent_after.specific_deferred.get_admin_display_title()
+                    }
                 }
-            }
-        )
+            )
+        # Page was moved
+        else:
+            PageLogEntry.objects.log_action(
```

Comment:
The two `log_action` calls are essentially the same, with the only thing changed being the action value..
suggestion: use `action = "wagtail.reorder" if parent_before.id == target.id else "wagtail.move"` and do a single `PageLogEntry.objects.log_action()` call

Reply:
Makes sense, will fix!